src/transformState.js

```diff
@@ -5,7 +5,29 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (let i = 0; i < actions.length; i++) {
+    switch (actions[i].type) {
+      case 'addProperties':
+        for (const keyAdd in actions[i].extraData) {
+          state[keyAdd] = actions[i].extraData[keyAdd];
+        }
```

Comment:
```suggestion
        Object.assign(state, actions[i].extraData);
```
You can assign all changes at once. No need for loop.

Reply:
Agree with Sergii.